.travis.yml

```diff
@@ -1,17 +1,12 @@
 language: php
 
 php:
-  - 5.4
   - 5.5
   - 5.6
 
 env:
-  - SYMFONY_VERSION=2.3.*
-  - SYMFONY_VERSION=2.4.*
-  - SYMFONY_VERSION=2.5.*
-  - SYMFONY_VERSION=2.6.*
```

Comment:
With the updated version-constraints in the composer.json, version 2.6, 2.7, 2.8 should be ok as well. No?!